src/main.cpp

```diff
@@ -1,13 +1,17 @@
-#include <psp2/kernel/processmgr.h>
-
 #include <iostream>
-#include <SDL2/SDL.h>
 
 #include "Util.h"
 #include "DMG.h"
+
+#ifdef VITA
+#include <psp2/kernel/processmgr.h>
+#include <SDL2/SDL.h>
```

Comment:
Needed to use different SDL.h includes for vita vs windows, not sure if there's a better way of setting SDL up but it seems minor so I'm going to leave it for now.